### PR TITLE
Mention assignee during line comments

### PR DESCRIPTION
**Pull request by illicitonion**

### CODE REVIEW

File: plugins/github.py [hunk 1]
```diff
@@ -361,14 +361,18 @@ def on_receive_pull_request_comment(self, data):
         pull_request = data["pull_request"]
         pr_username = pull_request["user"]["login"]
         pr_num = pull_request["number"]
+        assignee = "None"
```
Comment (kegsay) [inline plugins/github.py:364]: Did you mean for this to be a literal string?

Comment (illicitonion) [inline plugins/github.py:364]: Yep, a string literal placeholder for saying there's no assignee.

**Comment by kegsay** [comment]
LGTM

File: plugins/github.py (hunk 1 above)
Comment (kegsay) [inline plugins/github.py:364]: test comment

Comment (kegsay) [inline plugins/github.py:364]: another test

Comment (kegsay) [inline plugins/github.py:364]: this is probably binging you, sorry.

Comment (kegsay) [inline plugins/github.py:364]: one more for good luck

Comment (kegsay) [inline plugins/github.py:364]: I'm baaack.

File: plugins/github.py [hunk 2]
```diff
@@ -361,14 +361,18 @@ def on_receive_pull_request_comment(self, data):
         pull_request = data["pull_request"]
         pr_username = pull_request["user"]["login"]
```
Comment (kegsay) [inline plugins/github.py:362]: test

File: plugins/github.py [hunk 3]
```diff
@@ -361,14 +361,18 @@ def on_receive_pull_request_comment(self, data):
...
         pr_title = pull_request["title"]
         comment_url = data["comment"]["html_url"]
 
-        msg = "[<u>%s</u>] %s made a line comment on %s's <b>pull request #%s</b>: %s - %s" % (
+        msg = "[<u>%s</u>] %s made a line comment on %s's <b>pull request #%s</b> (assignee: %s): %s - %s" % (
             repo_name,
             username,
             pr_username,
             pr_num,
+            assignee,
```
Comment (kegsay) [inline plugins/github.py:375]: Ignore thie

File: plugins/github.py [hunk 4]
```diff
@@ -361,14 +361,18 @@ def on_receive_pull_request_comment(self, data):
...
+        if "assignee" in data["pull_request"]:
+            assignee = data["pull_request"]["assignee"]["login"]
         pr_title = pull_request["title"]
         comment_url = data["comment"]["html_url"]
 
-        msg = "[<u>%s</u>] %s made a line comment on %s's <b>pull request #%s</b>: %s - %s" % (
+        msg = "[<u>%s</u>] %s made a line comment on %s's <b>pull request #%s</b> (assignee: %s): %s - %s" % (
             repo_name,
             username,
             pr_username,
```
Comment (kegsay) [inline plugins/github.py:373]: Ignore me

File: plugins/github.py [hunk 5]
```diff
@@ -361,14 +361,18 @@ def on_receive_pull_request_comment(self, data):
         pull_request = data["pull_request"]
         pr_username = pull_request["user"]["login"]
         pr_num = pull_request["number"]
+        assignee = "None"
+        if "assignee" in data["pull_request"]:
+            assignee = data["pull_request"]["assignee"]["login"]
         pr_title = pull_request["title"]
         comment_url = data["comment"]["html_url"]
 
-        msg = "[<u>%s</u>] %s made a line comment on %s's <b>pull request #%s</b>: %s - %s" % (
```
Comment (kegsay) [inline plugins/github.py:367]: Ignore this comment.

File: plugins/github.py [hunk 6]
```diff
@@ -361,14 +361,18 @@ def on_receive_pull_request_comment(self, data):
         pull_request = data["pull_request"]
         pr_username = pull_request["user"]["login"]
         pr_num = pull_request["number"]
+        assignee = "None"
+        if "assignee" in data["pull_request"]:
+            assignee = data["pull_request"]["assignee"]["login"]
```
Comment (kegsay) [inline plugins/github.py:366]: Ignore that comment.

**Comment by kegsay** [comment]
test

File: plugins/github.py [hunk 7]
```diff
@@ -361,14 +361,18 @@ def on_receive_pull_request_comment(self, data):
         pull_request = data["pull_request"]
```
Comment (kegsay) [inline plugins/github.py:361]: test2

File: plugins/github.py (hunk 1 above)
Comment (kegsay) [inline plugins/github.py:364]: test 3

File: plugins/github.py [hunk 8]
```diff
@@ -361,14 +361,18 @@ def on_receive_pull_request_comment(self, data):
...
         pr_username = pull_request["user"]["login"]
         pr_num = pull_request["number"]
+        assignee = "None"
+        if "assignee" in data["pull_request"]:
```
Comment (kegsay) [inline plugins/github.py:365]: test 4

File: plugins/github.py (hunk 2 above)
Comment (kegsay) [inline plugins/github.py:362]: another

File: plugins/github.py [hunk 9]
```diff
@@ -361,14 +361,18 @@ def on_receive_pull_request_comment(self, data):
...
+            assignee = data["pull_request"]["assignee"]["login"]
         pr_title = pull_request["title"]
         comment_url = data["comment"]["html_url"]
```
Comment (kegsay) [inline plugins/github.py:368]: test0

**Comment by kegsay** [comment]
this is a test

File: plugins/github.py (hunk 1 above)
Comment (kegsay) [inline plugins/github.py:364]: test 4

File: plugins/github.py (hunk 9 above)
Comment (kegsay) [inline plugins/github.py:368]: test01

File: plugins/github.py [hunk 10]
```diff
@@ -361,14 +361,18 @@ def on_receive_pull_request_comment(self, data):
         pull_request = data["pull_request"]
         pr_username = pull_request["user"]["login"]
         pr_num = pull_request["number"]
```
Comment (kegsay) [inline plugins/github.py:363]: testing

File: plugins/github.py [hunk 11]
```diff
@@ -361,14 +361,18 @@ def on_receive_pull_request_comment(self, data):
...
+        assignee = "None"
+        if "assignee" in data["pull_request"]:
+            assignee = data["pull_request"]["assignee"]["login"]
         pr_title = pull_request["title"]
         comment_url = data["comment"]["html_url"]
 
-        msg = "[<u>%s</u>] %s made a line comment on %s's <b>pull request #%s</b>: %s - %s" % (
+        msg = "[<u>%s</u>] %s made a line comment on %s's <b>pull request #%s</b> (assignee: %s): %s - %s" % (
             repo_name,
             username,
```
Comment (kegsay) [inline plugins/github.py:372]: a test

**Comment by kegsay** [comment]
one more test

```
include blocks
```

and some `inline` **bold**

File: plugins/github.py (hunk 7 above)
Comment (kegsay) [inline plugins/github.py:361]: ping

**Comment by kegsay** [comment]
ping `test` foo **BAR**

**Comment by kegsay** [comment]
a tester

**Comment by kegsay** [comment]
another test

File: plugins/github.py (hunk 7 above)
Comment (kegsay) [inline plugins/github.py:361]: plugins/github.py

Comment (kegsay) [inline plugins/github.py:361]: a real test

File: plugins/github.py [hunk 12]
```diff
@@ -361,14 +361,18 @@ def on_receive_pull_request_comment(self, data):
         pull_request = data["pull_request"]
         pr_username = pull_request["user"]["login"]
         pr_num = pull_request["number"]
+        assignee = "None"
+        if "assignee" in data["pull_request"]:
+            assignee = data["pull_request"]["assignee"]["login"]
         pr_title = pull_request["title"]
```
Comment (kegsay) [inline plugins/github.py:367]: really?

**Comment by kegsay** [comment]
test one

File: plugins/github.py (hunk 7 above)
Comment (kegsay) [inline plugins/github.py:361]: reply test

File: plugins/github.py [hunk 13]
```diff
@@ -361,14 +361,18 @@ def on_receive_pull_request_comment(self, data):
...
         comment_url = data["comment"]["html_url"]
 
-        msg = "[<u>%s</u>] %s made a line comment on %s's <b>pull request #%s</b>: %s - %s" % (
+        msg = "[<u>%s</u>] %s made a line comment on %s's <b>pull request #%s</b> (assignee: %s): %s - %s" % (
             repo_name,
```
Comment (kegsay) [inline plugins/github.py:371]: line test

**Comment by kegsay** [comment]
test **two**

File: plugins/github.py (hunk 7 above)
Comment (kegsay) [inline plugins/github.py:361]: reply test 2

File: plugins/github.py [hunk 14]
```diff
@@ -361,14 +361,18 @@ def on_receive_pull_request_comment(self, data):
...
             username,
             pr_username,
             pr_num,
```
Comment (kegsay) [inline plugins/github.py:374]: test1

File: plugins/github.py (hunk 2 above)
Comment (kegsay) [inline plugins/github.py:362]: more test

**Pull request merged**